parts/windows/kuberneteswindowssetup.ps1

```diff
@@ -170,6 +170,9 @@ $global:ProvisioningScriptsPackageUrl = "{{GetVariable "windowsProvisioningScrip
 $global:WindowsPauseImageURL = "{{GetVariable "windowsPauseImageURL" }}";
 $global:AlwaysPullWindowsPauseImage = [System.Convert]::ToBoolean("{{GetVariable "alwaysPullWindowsPauseImage" }}");
 
+# Calico
+$global:WindowsCalicoPackageURL = "{{GetVariable "windowsCalicoPackageURL" }}";

```

Comment:
where is the variable windowsCalicoPackageURL assigned?

Reply:
In AKS as I described in this PR's description.